src/custom/hooks/useUSDCPrice/index.ts

```diff
@@ -186,7 +186,7 @@ export function useCoingeckoUsdValue(params: CoinGeckoUsdValueParams) {
   return useGetPriceQuote({ ...coingeckoUsdPrice, currencyAmount })
 }
 
-export function useHigherUSDValue(currencyAmount: CurrencyAmount<Currency> | undefined) {
+export function useHigherUSDValue(currencyAmount: CurrencyAmount<Currency> | undefined | null) {
```

Comment:
Do we really need to have both undefined and null? what is the meaning of each? 
Makes API of this hook harder imo

Reply:
no we don't made this for ease of typing as it's a bit random (Uni's fault hoenstly) but i can change these back good point